server/service/osquery_utils/queries.go

```diff
@@ -319,6 +319,11 @@ FROM logical_drives WHERE file_system = 'NTFS' LIMIT 1;`,
 		Platforms:  []string{"windows"},
 		IngestFunc: ingestDiskSpace,
 	},
+	"kubequery_info": {
+		Query:      `SELECT * from kubernetes_info`,
+		IngestFunc: ingestKubequeryInfo,
```

Comment:
Nit: 
```
Platforms: []{"linux"}
```
?
(To not send an extra query in the `distributed/read` to non-linux hosts.)

Reply:
Unfortunately we can't do this because of how kubequery works... It doesn't include most of the tables we normally expect, so we won't send this query if we set Platforms. It's the result of this query that actually allows us to set any platform at all.

In the future I think we should probably take the kubequery tables and just add them as Orbit extension tables -- I'm not a big fan of what they've done with "basequery" (basically just osquery with most of the tables removed and some other changes).